tests/conftest.py

```
@@ -60,8 +60,9 @@
 from pkdiagram.documentview import QmlEngine
 from pkdiagram.app import Application, AppController, Session as fe_Session, QmlUtil
 
-from fdserver.tests.conftest import *
-from fdserver.models import User
+from btcopilot.pro.models import User
+from btcopilot.tests.conftest import *
```

Ruff (0.13.2)

`from btcopilot.tests.conftest import *` used; unable to detect undefined names

(F403)